.bithoundrc

```diff
@@ -0,0 +1,58 @@
+{
+  "ignore": [
```

Comment:
It is the standard `.bithoundrc`, see http://blog.bithound.io/configuring-bithound-101/#ignoringfilesandfolders